<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTMLConverter</class>
 <widget class="QWidget" name="HTMLConverter">
  
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>295</height>
   </rect>
  </property>

  <property name="windowTitle">
   <string>HTML Converter</string>
  </property>

  <layout class="QGridLayout" name="gridLayout">
   

   // This is the first orizontal line
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
    
    // This represents the input box
     <item>
      <widget class="QLineEdit" name="path_line_edit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>/path/to/a/text/file</string>
       </property>
      </widget>
     </item>
   
    // This represents the browse button
     <item>
      <widget class="QPushButton" name="browse_btn">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
    
   </item>
    <item row="1" column="0" colspan="3">
      <item>
        <widget class="QTextEdit" name="result">
          <property name="readOnly">
            <bool>true</bool>
          </property>
          <property name="placeholderText">
            <string>HTML Output</string>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>700</width>
              <height>200</height>
            </size>
          </property>
        </widget>
      </item>

      <item>
      <widget class="QPushButton" name="convert_btn">
        <property name="text">
          <string>Convert</string>
        </property>
      </widget>
      </item>
    </item>
   
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>124</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   

   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>123</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>92</height>
      </size>
     </property>
    </spacer>
   
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
